iotlabcli/parser/main.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+
+# This file is a part of IoT-LAB cli-tools
+# Copyright (C) 2015 INRIA (Contact: admin@iot-lab.info)
+# Contributor(s) : see AUTHORS file
+#
+# This software is governed by the CeCILL license under French law
+# and abiding by the rules of distribution of free software.  You can  use,
+# modify and/ or redistribute the software under the terms of the CeCILL
+# license as circulated by CEA, CNRS and INRIA at the following URL
+# http://www.cecill.info.
+#
+# As a counterpart to the access to the source code and  rights to copy,
+# modify and redistribute granted by the license, users are provided only
+# with a limited warranty  and the software's author,  the holder of the
+# economic rights,  and the successive licensors  have only  limited
+# liability.
+#
+# The fact that you are presently reading this means that you have had
+# knowledge of the CeCILL license and that you accept its terms.
+
+"""Main parser."""
+
+import sys
+from argparse import ArgumentParser
+
+import iotlabcli.parser.auth
+import iotlabcli.parser.admin
+import iotlabcli.parser.experiment
+import iotlabcli.parser.node
+import iotlabcli.parser.profile
+import iotlabcli.parser.robot
+
+# from aggregation-tools
+try:
+    from iotlabaggregator.serial import main as aggregator_serial_main
+    from iotlabaggregator.sniffer import main as aggregator_sniffer_main
+    AGGREGATION_TOOLS = True
```

Comment:
Now that I'm reading this again, I think the global variable is not needed. You can play with the imported module:
```python
try:
    import iotlabaggregator
except ImportError:
   iotlabaggreagor = None
```

and then use `iotlabaggregator.serial.main` and `iotlabaggregator.sniffer.main` for the subcommands values. And check if iotlabaggregator is not None.
I think this will be simpler and it may fix the coverage issue.